usrsctplib/netinet/sctp_callout.c

```diff
@@ -55,22 +58,88 @@
 #include <netinet/sctp_pcb.h>
 #endif
 
+
+#if defined(__APPLE__) || defined(__Userspace__)
```

Comment:
Is it intentional that this change affects all userspace implementations and the kernel implementation on Mac OS?

Reply:
I indented to modify only userspace implementation. Will fix that.